site/data/evangelists.yaml

```diff
@@ -176,18 +164,17 @@
   has_training: Y
   notes: >
     toolsmith author, infosec tools evangelist.
-    <a href="https://holisticinfosec.org/research-mainmenu-45/22-research/193-owasp-top-10-tools-tactics" rel="nofollow">OWASP Top 10 tools, tactics</a> 
+    <a href="https://resources.infosecinstitute.com/owasp-top-10-tools-and-tactics/" rel="nofollow">OWASP Top 10 tools, tactics</a> 
     presented at SANSFIRE, <code>SecureWorld</code>, 
     ISSA International, etc.
 
 - name: Azzeddine Ramrami
   email: azzeddine.ramrami@owasp.org
-  linkedin: http://fr.linkedin.com/pub/azzeddine-ramrami/5/b7a/107/
```

Comment:
https://www.linkedin.com/in/%D8%B1%D9%85%D8%B1%D8%A7%D9%85%D9%8A-%D8%B9%D8%B2%D8%A7%D9%84%D8%AF%D9%8A%D9%86-azzeddine-ramrami-107b7a5/